### PR TITLE
Thumbnail

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
Quick review please! 
There are two things that I am not entirely sure about from an organizational point of view.
1. In the delte_upload/1 function in content.ex I am using the same function for both deleting the video file and for deleting the associated thumbnail. I chose this path, as a delete_upload_file and delete_thumbnail function would be identical. 

2. Should I create a create_thumbnail function in my content.ex file that then calls the one in the Thumbnailer? I am asking because this is content-related and - in terms of consisteny - I also store the video file via the store_file function in content.ex.

Thank you!

### CODE REVIEW

File: lib/the_little_thinkers_space/upload_paths_helper.ex [hunk 1]
```diff
@@ -36,4 +36,36 @@ defmodule TheLittleThinkersSpace.UploadPathsHelper do
...
+  def thumbnail_path(path) do
+    case is_binary(path) do
+      true ->
+        extension = Path.extname(path)
+        thumbnail_path = String.replace(path, extension, ".jpg")
+        {:ok, thumbnail_path}
+
+      _ ->
+        {:error, :no_thumbnail_path}
+    end
```
Comment (alanvardy) [inline lib/the_little_thinkers_space/upload_paths_helper.ex]:
```suggestion
    case path do
      path when is_binary(path) ->
        extension = Path.extname(path)
        thumbnail_path = String.replace(path, extension, ".jpg")
        {:ok, thumbnail_path}

      _ ->
        {:error, :no_thumbnail_path}
    end
```

Comment (alanvardy) [reply lib/the_little_thinkers_space/upload_paths_helper.ex]: Or even better, use 2 function heads

File: lib/the_little_thinkers_space/upload_paths_helper.ex [hunk 2]
```diff
@@ -36,4 +36,36 @@ defmodule TheLittleThinkersSpace.UploadPathsHelper do
...
+          path
+          |> String.replace("app/lib/the_little_thinkers_space-0.1.0/", "")
+          |> String.replace("priv/static/", "/")
+          |> String.replace("//", "/")
+
+        {:ok, thumbnail_show_path}
+
+      _ ->
+        {:error, :no_thumbnail_show_path}
+    end
```
Comment (alanvardy) [inline lib/the_little_thinkers_space/upload_paths_helper.ex]: If it is not a binary, and not nil, what else can it be?

File: priv/repo/migrations/20220308150127_add_not_null_constraint_to_path.exs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+defmodule TheLittleThinkersSpace.Repo.Migrations.AddNotNullConstraintToPath do
+  use Ecto.Migration
+
+  def change do
+    alter table(:uploads) do
+      modify :path, :string, null: false, from: :string
```
Comment (alanvardy) [inline priv/repo/migrations/20220308150127_add_not_null_constraint_to_path.exs:6]: Nice

**Review by alanvardy — APPROVED** [review]

File: lib/the_little_thinkers_space/upload_paths_helper.ex (hunk 1 above)
Comment (CorneliaKelinske) [reply lib/the_little_thinkers_space/upload_paths_helper.ex]: Coolio! Done thank you!

File: lib/the_little_thinkers_space/upload_paths_helper.ex (hunk 2 above)
Comment (CorneliaKelinske) [reply lib/the_little_thinkers_space/upload_paths_helper.ex]: Is this a trick question? lol I'm going with "nothing" -> I removed this option completely

**Pull request merged**